catalog/app/components/JsonEditor/JsonEditor.js

```diff
@@ -0,0 +1,101 @@
+import * as R from 'ramda'
+import * as React from 'react'
+import * as M from '@material-ui/core'
+
+import Column from './Column'
+import Errors from './Errors'
+import State from './State'
+
+const useStyles = M.makeStyles((t) => ({
+  inner: {
+    display: 'flex',
+    overflowX: 'auto',
+    '-ms-overflow-style': 'none',
```

Comment:
a note for the future: in JSS this can be written as `MsOverflowStyle`. the general rule is it inserts a dash before and lowers every uppercase letter.